iot_verifier/src/reward_share.rs

```diff
@@ -61,8 +99,11 @@ impl GatewayShare {
             r#"
             insert into gateway_shares (hotspot_key, reward_type, reward_timestamp, hex_scale, reward_unit, poc_id)
             values ($1, $2, $3, $4, $5, $6)
-            on conflict (hotspot_key, poc_id) do update set
-            reward_type = EXCLUDED.reward_type, reward_timestamp = EXCLUDED.reward_timestamp, hex_scale = EXCLUDED.hex_scale, reward_unit = EXCLUDED.reward_unit
+            on conflict (poc_id) do update set
```

Comment:
```suggestion
            on conflict (hotspot_key, poc_id) do update set
```
i think this has to stay a conflict check against the entire `(hotspot_key, poc_id)` composite key because it's valid for two hotspots to have reward records in the database with with the same `poc_id` within the same reward period, one being for beaconing and any number for witnessing. because of that, this table also uses the composite of the hotspot key and the poc id as the primary key so not checking against both could inadvertently cause a collision if we did attempt to insert a "duplicate" record that didn't have a conflict resolution defined

Reply:
darn, i was meant to put this entire sql sequence back to its original form.  Will fix...

Reply:
addressed here: 95abec833a8dc0eba6ff9cf1fe4c961054d88d5a